pkg/services/grpcserver/service.go

```diff
@@ -68,21 +68,27 @@ func ProvideService(cfg *setting.Cfg, features featuremgmt.FeatureToggles, authe
 		}
 	}
 
+	unaryInterceptors := []grpc.UnaryServerInterceptor{
+		interceptors.LoggingUnaryInterceptor(s.logger, s.cfg.EnableLogging), // needs to be registered after tracing interceptor to get trace id
+		middleware.UnaryServerInstrumentInterceptor(grpcRequestDuration),
+	}
+	streamInterceptors := []grpc.StreamServerInterceptor{
+		interceptors.TracingStreamInterceptor(tracer),
```

Comment:
Please rebase on latest `main`, we have recently added logging stream interceptor here.